### DISCUSSION TITLE
modal gallery using classic editor

### DISCUSSION THREAD

**Discussion by noyloza** [bootscore/bootscore#242] [General]
Is it possible to use the gutenberg lightbox gallery plugin for bootscore without using the gutenberg editor? I'm using custom fields or just coding directly onto the php page. How could I make it work? Any ideas greatly appreciated.

**Comment by crftwrk** [comment]
Unfortunately not because it is a plugin for the default Gutenberg Gallery Block. If you disable the blocks with the classic editor then is there no gallery block anymore. 

But you can try any other gallery plugin which can be used by shortcode. https://wordpress.org/plugins/search/gallery/
